jplag.endToEndTesting/src/main/java/de/jplag/endToEndTesting/helper/JsonHelper.java

```diff
@@ -0,0 +1,22 @@
+package de.jplag.endToEndTesting.helper;
+
+import java.nio.file.Paths;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import model.ResultJsonModel;
+
+public class JsonHelper {
+
+	public static List<ResultJsonModel> getResultModelFromPath(String pathToJsonFile)
+			throws JsonMappingException, JsonProcessingException, Exception {
+		ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Make it static and attribute

Reply:
remove unnecessary variables in the method call and directly use the constant path variables